sci-electronics/drahnr-oregano/drahnr-oregano-0.84.39.ebuild

```diff
@@ -0,0 +1,77 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+PYTHON_COMPAT=( python3_{4,5,6} )
+PYTHON_REQ_USE='threads(+)'
+
+inherit gnome2-utils python-single-r1 waf-utils xdg-utils
```

Comment:
If you don't install anything using Python, python-single-r1 is not the correct eclass to use. See https://wiki.gentoo.org/wiki/Project:Python/Eclasses

Reply:
as I understand  waf-utils need call python-single-r1_pkg_setup or I incorrect imagine how it work

Reply:
python-any-r1